src/constants.ts

```diff
@@ -57,7 +56,7 @@ const FormLoginData = {
   Email: {
     TextError: 'Введите правильный Email!',
     RegEx:
-      /^[a-z0-9-]+(?:\.[a-z0-9-]+)*@(?:[a-z0-9](?:[a-z-]*[a-z])?\.)+[a-z]{2,4}$/,
+      /^[A-Za-z0-9-]+(?:\.[A-Za-z0-9-]+)*@(?:[A-Za-z0-9](?:[A-Za-z-]*[a-z])?\.)+[a-z]{2,4}$/,
```

Comment:
```suggestion
      /^[a-z0-9-]+(?:\.[a-z0-9-]+)*@(?:[a-z0-9](?:[a-z-]*[a-z])?\.)+[a-z]{2,4}$/i,
```

https://regex101.com/